frontend/javascripts/oxalis/controller/viewmodes/plane_controller.tsx

```diff
@@ -154,6 +167,19 @@ class VolumeKeybindings {
       },
     };
   }
+
+  static getExtendedKeyboardControls() {
+    return {
+      b: () => setTool(AnnotationToolEnum.BRUSH),
+      e: () => setTool(AnnotationToolEnum.ERASE_BRUSH),
+      l: () => setTool(AnnotationToolEnum.TRACE),
+      r: () => setTool(AnnotationToolEnum.ERASE_TRACE),
+      f: () => setTool(AnnotationToolEnum.FILL_CELL),
+      p: () => setTool(AnnotationToolEnum.PICK_CELL),
+      q: () => setTool(AnnotationToolEnum.QUICK_SELECT),
+      o: () => setTool(AnnotationToolEnum.PROOFREAD),
```

Comment:
@dieknolle3333 In order to define new "extended" shortcuts, just define them as regular keyboard shortcuts leaving out the `ctrl + k`. But make sure these are included in their own object that is additionally passed to the `InputKeyboardNoLoop` in the constructor. E.g. `ctrl + k, 1` becomes simply `1`.


Then you need to create the `InputKeyboardNoLoop` like this: 
```javascript
const extendedKeyboardBindings = { 1: () -> setBrushToPreset(PRESETS.1)};
new InputKeyboardNoLoop(normalKeyboardBindings, {options}, extendedKeyboardBindings);
```
This way pressing `ctrl + k, 1`  should call ` () -> setBrushToPreset(PRESETS.1)`